app/utils/simplified-account/validation/organisation-details.schema.js

```diff
@@ -0,0 +1,74 @@
+const { body } = require('express-validator')
+const { invalidTelephoneNumber } = require('@utils/telephone-number-utils')
+const { isValidUrl } = require('@utils/validation/server-side-form-validations')
+const { postcodeRegex } = require('@utils/validation/postcode-validation')
+
+const ORGANISATION_NAME_MAX_LENGTH = 100
+const ADDRESS_FIELD_MAX_LENGTH = 255
+
+const organisationDetailsSchema = {
+  organisationName: {
+    validate: body('organisationName').trim()
+      .notEmpty()
+      .withMessage('Enter an organisation name')
+      .bail()
+      .isLength({ max: ORGANISATION_NAME_MAX_LENGTH })
+      .withMessage(`Organisation name must be ${ORGANISATION_NAME_MAX_LENGTH} characters or fewer`)
+  },
+  organisationAddress: {
+    line1: {
+      validate: body('addressLine1').trim()
+        .notEmpty()
+        .withMessage('Enter a building and street')
+        .bail()
+        .isLength({ max: ADDRESS_FIELD_MAX_LENGTH })
+        .withMessage(`Building and street must be ${ADDRESS_FIELD_MAX_LENGTH} characters or fewer`)
+    },
+    line2: {
+      validate: body('addressLine2').trim()
+        .isLength({ max: ADDRESS_FIELD_MAX_LENGTH })
+        .withMessage(`Building and street must be ${ADDRESS_FIELD_MAX_LENGTH} characters or fewer`)
+    },
+    city: {
+      validate: body('addressCity').trim()
+        .notEmpty()
+        .withMessage('Enter a town or city')
+        .bail()
+        .isLength({ max: ADDRESS_FIELD_MAX_LENGTH })
+        .withMessage(`Town or city must be ${ADDRESS_FIELD_MAX_LENGTH} characters or fewer`)
+    },
+    postcode: {
+      validate: body('addressPostcode').trim()
+        .notEmpty()
+        .withMessage('Enter a postcode')
+        .bail()
+        .matches(postcodeRegex)
+        .withMessage('Enter a real postcode')
```

Comment:
If we allow other country addresses, why do we use UK postcode validation?

Reply:
I'm not actually sure on the answer to this, I think actually I may need to modify it slightly as the existing code would apply the UK postcode validation if the chosen country is the United Kingdom, otherwise just accept whatever postcode was entered. Perhaps I should replicate that instead

Reply:
yes, probably safest to replicate how it works now, could approve on that basis but perhaps refer to product/design to make sure they are happy with that